.github/CONTRIBUTING.md

```diff
@@ -1,16 +1,16 @@
-[<img src="https://avatars1.githubusercontent.com/u/30810402?s=460&v=4" width="100px;"/><br /><sub>Arnav Borborah</sub>](https://github.com/arnavb)<br /> | [<img src="https://avatars2.githubusercontent.com/u/28451366?s=400&v=4" width="100px;"/><br /><sub>Yatharth Shah</sub>](https://github.com/yatharthshahjpr)<br /> | [<img src="https://avatars2.githubusercontent.com/u/16862997?v=4" width="100px;"/> <br /><sub>David Wu</sub>](https://github.com/Pl4gue)<br /> | [<img src="https://avatars3.githubusercontent.com/u/16002727?v=4" width="100px;"/> <br /><sub>Mohit Khare</sub>](https://github.com/mkfeuhrer)<br /> | [<img src="https://avatars0.githubusercontent.com/u/14811383?v=4" width="100px"/><br /><sub>Saif Ahmad</sub>](https://github.com/the-saif-ahmad)<br /> | [<img src="https://avatars0.githubusercontent.com/u/22833885?v=4" width="100px"/><br /><sub>Sanchit Jalan</sub>](https://github.com/Sanchit-20)<br /> | 
-[<img src="https://avatars3.githubusercontent.com/u/4724181?v=4" width="100px;"/><br /><sub>Yogesh Badhe</sub>](https://github.com/warlockz)<br /> |
-<!-- ALL-CONTRIBUTORS-LIST:END -->
+`“ Getting started is the most difficult thing to do; once you file it out, they rest of the journey is as soft as the straw. Be a good beginner. ” 
+   ― Israelmore Ayivor`
+ 
+ 1. You can start contributing by selecting any field from our high-level structure that intrests you the most.
+ 2. Once you have chosen the path, **make sure that the algorithm is not already implemented**. This step is very important because we dont't want your time and energy to be go in vain.
+ 3. When you complete step 2, read the [style guides](https://github.com/OpenGenus/cosmos/tree/master/guides/coding_style) for the language you want to implement. The coding style for every document is required to be similar so that it can be processed quickly and precisely across all platforms. 
+ 4. Last but not the least, follow the [code of conduct](https://github.com/OpenGenus/cosmos/blob/master/.github/CODE_OF_CONDUCT.md).
+ 
+ So go on and start contributing. :smile: May the code be with you :metal:
```

Comment:
What are you waiting for? Go on and start contributing! :smile: May the code be with you. :metal: